src/fieldValidators.js

```diff
@@ -86,7 +86,7 @@ const testEmailFormat = (value: ?string) => {
 const testLinkedInUrlFormat = (value: ?string) => {
     if (!value) return null;
 
-    const regexp = /(https?)?:?(\/\/)?(([w]{3}||\w\w)\.)?linkedin.com\/in\//; // eslint-disable-line no-control-regex
+    const regexp = /(https?):(\/\/)(([w]{3}||\w\w)\.)?linkedin.com\/in\/([a-zA-Z0-9-])+/; // eslint-disable-line no-control-regex
```

Comment:
<img width="701" alt="Capture d’écran 2021-10-04 à 14 40 33" src="https://user-images.githubusercontent.com/25125613/135853354-0b00fe76-682b-4e16-98b5-06753170cc33.png">
<img width="701" alt="Capture d’écran 2021-10-04 à 14 41 44" src="https://user-images.githubusercontent.com/25125613/135853361-4c39d7d0-8036-4e57-9973-5b713573da9f.png">

According to this regex simulator https://regex101.com/

Special characters in URL are not accepted. Take my profile LinkedIn as an example : https://www.linkedin.com/in/%E2%9A%9B-mohamed-g-716a96119/

Reply:
Thank ! I missed this case ! 

It's done.